step-controller-plugins/step-controller-plugins-timeseries/src/main/java/step/plugins/timeseries/TimeSeriesService.java

```diff
@@ -119,6 +126,7 @@ public AsyncTaskStatus<Object> rebuildTimeSeries(TimeSeriesRebuildRequest reques
                         count.increment();
                         timeSeriesBucketingHandler.ingestExistingMeasurement(measurement);
                     });
+                    ingestionPipeline.flush();
```

Comment:
@lucianbaciu The TimeSeriesIngestionPipeline should be closed in a finally block to avoid any leak